README.rst

```diff
@@ -136,7 +154,7 @@ A collection is where records are stored.
     client.get_collection('receipts', bucket='payments')
 
     # To delete an existing collection.
-    client.delete_collection('receipts', bucket='payments')
+    client.delete_collection('receipts', bucket='payments', if_exists=True)
```

Comment:
I would rather add a not telling that if we want to fail silently we can add `if_exists=True`